docs/release-notes/index.rst

```diff
@@ -3,9 +3,9 @@
 Release notes
```

Comment:
delete the deprecated features and remove the 1.0, unless our first release is a 1.0, So maybe remove it all, @medubelko ?

Reply:
I think it would help to keep the template in code for future use. If we want to remove the page from rendering and wait for the initial release, let's do that instead (list it in `exclude_patterns` in `docs/conf.py`).